### PR TITLE
[VOQHLD] Update for cli commands

### PR DESCRIPTION
**Pull request by vganesan-nokia**
The CLI section is updated for the design of show commands for system ports and system neighbors

### CODE REVIEW

**Comment by anshuv-mfst** [comment]
@ ysmanman - could you please review, thanks.

File: doc/voq/voq_hld.md [hunk 1]
```diff
@@ -477,7 +477,99 @@ The system port configuration has the parameters listed below.
...
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
+```
+### 2.7.1 System ports show command
+The information for the system ports are taken from the **SYSTEM_PORT_TABLE** of **APPL_DB** from a specific namespace
```
Comment (ysmanman) [inline doc/voq/voq_hld.md:501]: Is this show cmd to dump config or status of system port? If it dumps status, should we dump from chassis_app_db?

Comment (vganesan-nokia) [reply doc/voq/voq_hld.md:501]: This command is to show the config only.

**Review by ysmanman — APPROVED** [review]

File: doc/voq/voq_hld.md [hunk 2]
```diff
@@ -477,7 +477,99 @@ The system port configuration has the parameters listed below.
...
+The information for the system neighbors are taken from the **SYSTEM_NEIGH_TABLE** of **CHASSIS_APP_DB** in the supervisor card. 
+#### Syntax
+```
+admin@Linecard2:~$ show chassis system-neighbors -h
+Usage: show chassis system-neighbors [OPTIONS]
+
+  Show VOQ system neighbors information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
```
Comment (arlakshm) [inline doc/voq/voq_hld.md:552]: Same as above. Do we need the `-d` option ?

**Review by arlakshm — COMMENTED** [review]
as comments

File: doc/voq/voq_hld.md [hunk 3]
```diff
@@ -477,7 +477,99 @@ The system port configuration has the parameters listed below.
...
+The information for the system ports are taken from the **SYSTEM_PORT_TABLE** of **APPL_DB** from a specific namespace 
+#### Syntax
+```
+admin@Linecard2:~$ show chassis system-ports -h
+Usage: show chassis system-ports [OPTIONS] [SYSTEMPORTNAME]
+
+  Show VOQ system ports information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
```
[the remark this answers is not in the document]
Comment (vganesan-nokia) [reply doc/voq/voq_hld.md:510]: Yes. I'll remove the -d option.

File: doc/voq/voq_hld.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vganesan-nokia) [reply doc/voq/voq_hld.md:552]: I'll remove this.

Comment (vganesan-nokia) [reply doc/voq/voq_hld.md:552]: No. We do not need this. I'll remove this.

File: doc/voq/voq_hld.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vganesan-nokia) [reply doc/voq/voq_hld.md]: Fixed. Removed -d option. Also namespace option is made "required" as suggested the code PR.

File: doc/voq/voq_hld.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vganesan-nokia) [reply doc/voq/voq_hld.md]: Fixed. Removed the "-d" option.

Comment (vganesan-nokia) [reply doc/voq/voq_hld.md]: Fixed. Removed "-d" option.

**Review by arlakshm — APPROVED** [review]

**Pull request merged**